### PR TITLE
Cart management branch

### PR DESCRIPTION
**Pull request by ruchiragoswami**
### Cart Management 
Contains:
1. Product card
2. Price details

cart-PR

### CODE REVIEW

**Review by Husain01 — APPROVED** [review]

File: other-pages/cart.html [hunk 1]
```diff
@@ -1,14 +1,147 @@
...
+            </div>
+
+            <div class="button-holder">
+                <button class="btn btn-primary-outlined btn-for-hover custom-button"> Logout </button>
+            </div>
+
+            <div class="icon-holder">
+                <a href="./wishlist.html"> <span class="material-icons-outlined custom-icon"> favorite_border </span>
+                    <!-- <span class="material-icons-outlined">
+                        exposure_zero                     </span>  -->
```
Comment (AtulPant2704) [inline other-pages/cart.html]: If you don't need these lines you can delete them instead of just commenting it.

File: other-pages/cart.html [hunk 2]
```diff
@@ -1,14 +1,147 @@
...
+
+            <div class="icon-holder">
+
+                <a href="#0">
+                    <span class="material-icons-outlined custom-icon cart-icon">
+                        shopping_cart_border
+                    </span>
+                    <!-- <span class="material-icons-outlined">
+                        exposure_zero
+                    </span> -->
```
Comment (AtulPant2704) [inline other-pages/cart.html]: If you don't need these lines you can delete them instead of just commenting it.

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

**Review by AnkurChunekar — APPROVED** [review]

File: styles/cart.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    margin-top: 2rem;
+}
+
+.table-order-details>caption {
+    border-bottom: 2px solid var(--primary-color);
+    margin-bottom: 1rem;
+}
+
+.table-order-details>tbody>tr:nth-child(4)>th,
+.table-order-details>tbody>tr:nth-child(4)>td {
```
Comment (AtulPant2704) [inline styles/cart.css:87]: I think you should have a space between > selector as it would be much more easy to read.

**Review by AtulPant2704 — APPROVED** [review]
Everything else looks good

File: styles/cart.css (hunk 3 above)
Comment (ruchiragoswami) [reply styles/cart.css:87]: I agree, but 'beautify' extension in VS code removes the spaces between selectors, even if I add the spaces manually.

**Pull request merged**